File: BlogHS.Client/src/app/models/Entry.ts
export default class Entry {
    constructor() {
        this.id = 0;
        this.title = '';
        this.content = '';
        this.thumbnailUrl = '';
        this.creationDate = '';
    }
    id?: number;
    title: string;
    content: string;
    thumbnailUrl: string;
    creationDate: string;
}

File: BlogHS.Domain/Entities/Entry/Entry.cs
using BlogHS.Domain.Base;
using System.ComponentModel.DataAnnotations;

namespace BlogHS.Domain.Entities.Entry
{
    public partial class Entry : BaseEntity<int>
    {
        [Required]
        public string? Title { get; set; }
        [Required]
        public string? Content { get; set; }
        [Required]
        public string? ThumbnailUrl { get; set; }
        [Required]
        public string? ThumbnailContent { get; set; }
        [Required]
        public DateTimeOffset? CreationDate { get; set; }
        public DateTimeOffset? ModificationDate { get; set; }
    }
}